docs/multi-environment-workflow.md

```diff
@@ -0,0 +1,94 @@
+# Incorporating into multi-environment workflows
+
+The Deploy CLI is most powerful when working within a multi-tenant, multi-environment context. When integrated into your CI/CD development workflows, can be used to propagate Auth0 changes from feature development all the way through production.
```

Comment:
```suggestion
The Deploy CLI supports working within a multi-tenant, multi-environment context. When integrated into your CI/CD development workflows, can be used to propagate Auth0 changes from feature development all the way through production.
```

IMO the wording implies that it shouldn't be used for anything else than multi env context.